client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.controller.js

```diff
@@ -1,209 +1,278 @@
+                    });
+
+                    if (isTrunk) {
+                        apiResources.getSimultaneousLines = OvhApiOrderTelephony.Lexi().getSimultaneousTrunkLines;
+                        apiResources.orderSimultaneousLines = OvhApiOrderTelephony.Lexi().orderSimultaneousTrunkLines;
+                    }
+
+                    self.saved = angular.copy(self.options);
+                    return self.options;
+                }),
+
+                OvhApiTelephony.Line().Lexi().maximumAvailableSimultaneousLines({
+                    billingAccount: $stateParams.billingAccount,
+                    serviceName: $stateParams.serviceName
+                }).$promise.then(function (maximumAvailableSimultaneousLines) {
+                    self.options.maximumAvailableSimultaneousLines = maximumAvailableSimultaneousLines.maximum;
+                    return self.options;
+                })
+
+            ]).finally(function () {
+                self.loading.init = false;
+            }).catch(function () {
+                return $translate("telephony_line_actions_line_calls_simultaneous_line_load_error").then(function (message) {
+                    return Toast.error(message);
+                });
+            });
         }
 
-        return $q.when(null);
-    }, 200, false);
+        init();
 
-    function init () {
+        /* ===========================
+        =            BULK            =
+        ============================ */
 
-        self.loading = {
-            init: true,
-            order: false,
-            doOrder: false,
-            save: false
-        };
-        self.contractsAccepted = false;
-        self.contracts = null;
-        self.prices = null;
-        self.saved = { };
-        self.offerTasks = [];
-        self.options = {
-            simultaneousLines: null,
-            maximumAvailableSimultaneousLines: 0,
-            minimumAvailableSimultaneousLines: 1
+        self.bulkDatas = {
+            billingAccount: $stateParams.billingAccount,
+            serviceName: $stateParams.serviceName,
+            infos: {
+                name: "simultaneousCalls",
+                actions: [{
+                    name: "removeSimultaneousLines",
+                    route: "/telephony/{billingAccount}/line/{serviceName}/removeSimultaneousLines",
+                    method: "POST",
+                    params: null
+                }, {
+                    name: "addSimultaneousLines",
+                    route: "/order/telephony/lines/{serviceName}/addSimultaneousLines",
+                    method: "POST",
+                    params: null
+                }]
+            }
         };
 
-        return $q.all([
+        self.filterServices = function (services) {
+            return _.filter(services, function (service) {
+                return ["sip", "mgcp"].indexOf(service.featureType) > -1 && service.hasValidPublicOffer();
```

Comment:
There is an other subtlety on this feature. For trunks, API call is not the same. So you should have an error when you select a trunk. As simultaneous calls order i is not working very well for the moment (there is the notion of simultaneous calls pack that is missing), you need to disable the bulk action on trunks.
The easiest way to know if the sip line is a trunk is to use either `getRealFeatureType` method of `VoipServiceLine` factory or `isSipTrunk` method of the same factory.

Reply:
Also : please rebase your branch and fix conflicts ;-)